facade/src/main/java/org/jboss/pnc/facade/executor/BuildExecutorTriggerer.java

```diff
@@ -94,7 +94,10 @@ public BuildExecutionSession executeBuild(
                         .getBpmTaskByBuildTaskId(bpmManager, statusChangedEvent.getBuildTaskId());
 
                 if (bpmTask.isPresent()) {
```

Comment:
I think this whole block is redundant as the final result is sent by bpmNotifier.sendBuildExecutionCompleted

Reply:
Ok I was worried that since only statusChangedEvent with buildResults are sent by bpmNotifier.sendBuildExecutionCompleted, we could have missed some notifications. But, looking at DefaultBuildExecutionSession.setStatus, every DefaultBuildExecutorStatusChangedEvent which is final, has buildResults set. So I think that yes we are fine to remove it.

Reply:
@matejonnet OK I have pushed the changes and made some cleanup